config.toml

```diff
@@ -10,12 +10,24 @@ download = "/tmp/vimana/celestia/init.sh"
 binary = "/tmp/vimana/celestia/celestia"
 download = "/tmp/vimana/celestia/init.sh"
 
+
 [components.avail]
 
 [components.avail.light]
 binary = "avail-light"
 download = "/tmp/vimana/avail/init.sh"
 
+
+[components.gmworld]
+
+[components.gmworld.da]
+binary = "gmworld-da"
+download = "/home/ankit/vimana/scripts/rollup_init.sh"
```

Comment:
let's move these to `~/.vimana/gmd/scripts/rollup_init.sh` so we can also move other scripts to their appropriate directories.

Reply:
done